### PR TITLE
Handle a whole slough of new squirely releng messages.

### PR DESCRIPTION
**Pull request by ralphbean**
They all come from this script:

https://pagure.io/releng/blob/master/f/scripts/make-updates

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/compose.py [hunk 1]
```diff
@@ -76,9 +118,19 @@ def link(self, msg, **config):
         else:
             if 'epelbeta' in msg['topic']:
                 base = "https://dl.fedoraproject.org/pub/epel/beta/7"
-            else:
+            elif 'branched' in msg['topic'] or 'rawhide' in msg['topic']:
                 base = "https://dl.fedoraproject.org/pub/" + \
                     "fedora/linux/development"
+            else:
+                branch = msg['msg'].get('branch', 'not-a-number')
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/compose.py]: is this rather 'not-a-branch'?

**Comment by pypingou** [comment]
:+1: but looks a little weak on tests, including if the branch isn't specified or is not a number

File: fedmsg_meta_fedora_infrastructure/compose.py (hunk 1 above)
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/compose.py]: No - "not-a-number", because I try to cast it to an ``int`` just two lines below.

Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/compose.py]: The not-a-number could be "rawhide", for instance.  ;)

**Comment by ralphbean** [comment]
As far as I can tell, that make-updates script always provides the branch and it is always a number.  If it is absent or not a number, I think it will fail back to the other code paths just fine (and the other tests test those code paths).  Thanks @pypingou!

File: fedmsg_meta_fedora_infrastructure/compose.py (hunk 1 above)
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/compose.py]: But then ``msg['msg'].get('branch')`` would return `rawhide` no?

**Pull request merged**